hypothesis-python/tests/cover/test_stateful.py

```diff
@@ -97,11 +97,11 @@ def execute_step(self, step):
 class BalancedTrees(RuleBasedStateMachine):
     trees = u'BinaryTree'
```

Comment:
nit: It would probably make more sense to change this to `trees = Bundle(u'BinaryTree')` and leave the various references to trees unchanged.